ADB_world/Worldmap_month/Data_frames.py

```diff
@@ -0,0 +1,128 @@
+########################################
+# IMPORTS #############################
+#######################################
+
+import json
+import pandas as pd
+import geopandas as gpd
+import os
+from shapely.geometry import LineString
+
+
+########################################
+# load data ############################
+########################################
+
+# Get the current directory
+current_directory = os.path.dirname(os.path.realpath(__file__))
+
+# Read healtcheck file to get all airports with informations
+file_path = os.path.join(current_directory, "Airport Data/Airports.json")
+with open(file_path, 'r') as f:
+    airports_icao = json.load(f)
+
+
+########################################
+# adjust data ##########################
+########################################
+
+# Extract icao names of airports from airports_icao and create a DataFrame
+airport_list = [{'icao': airport_loc} for airport_loc in airports_icao['items']]
+airport_list_df = pd.DataFrame(airport_list)
+
+# Create a new DataFrame to store 'icao', 'airport_name', lat', and 'lon' of each airport
+airport_info_list = []
+for icao_airport in airport_list_df['icao']:
+    file_path = os.path.join(current_directory, f"Airport Data/airports_info/{icao_airport}.json")
+    with open(file_path, 'r') as f:
+        airport_info = json.load(f)
+    airport = {
+            'icao': icao_airport,
+            'airport_name': airport_info['fullName'],
+            'lat': airport_info['location']['lat'],
+            'lon': airport_info['location']['lon'],
+        }
+    airport_info_list.append(airport)  # appends it to list
+airports_info_df = pd.DataFrame(airport_info_list)
+
+# Create a new GeoDataFrame to store 'icao', 'airport_name', 'lat', and 'lon', 'geometry', of each airport (only needed if plotting is done using Geopanda)
+departure_airports_geodf = gpd.GeoDataFrame(
+        airports_info_df,
+        geometry=gpd.points_from_xy(
+            x=airports_info_df["lon"],
+            y=airports_info_df["lat"],
+            # Specify the coordinate reference system (standard for lat/lon)
+            crs='EPSG:4326'
+        )
+    )
+
+
+########################################
+# create one big dataframe #############
```

Comment:
Here I will need a descriptive docstring. I don't want step through the entire code to figure out what the output looks like. You can take my functions here for inspiration: https://github.com/sustainableaviation/EcoPyLot/blob/8a6b7b5dff270293a38754459cb5f2812471245e/ecopylot/inout.py#L57